data/features/css-autofill.js

```diff
@@ -0,0 +1 @@
+module.exports={A:{D:{"33":"0 1 2 3 4 5 6 7 8 9 undefined lB cB I e J E F G A B C K L D M N O f g h i j k l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T U V W X Y Z a b c R d H fB pB"},L:{"33":"0 1 2 3 4 5 6 7 8 9 O l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T U V W X Y Z a b c R d H fB pB"},B:{"33":"P Q T U V W X Y Z a b c R d H fB pB","34":"C K L D M N O"},C:{"1":"X Y Z a b c R d H fB pB qB","2":"0 1 2 3 4 5 6 7 8 9 undefined lB cB I e J E F G A B C K L D M N O f g h i j k l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T mB U V W nB oB"},M:{"1":"X Y Z a b c R d H fB pB qB undefined","2":"0 1 2 3 4 5 6 7 8 9 I e J E F G A L D M N O f g h i j k l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB P Q T mB U V W"},A:{"2":"undefined lB cB I e J E F G A B kB"},F:{"33":"0 1 2 3 4 5 6 7 8 9 D M N O f g h i j k l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB JB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T","34":"lB cB I e J E F G A B C nB oB sB uB undefined vB ZC hB 1B 2B aB iB 3B bB"},K:{"33":"1 2 3 4 5 6 7 8 9 L D M O f g h i k l m n o p q s t u v w x AB BB CB DB EB FB GB HB IB dB JB eB KB LB S","34":"B C hB aB iB bB"},E:{"33":"undefined cB I e J E F G A B C K L D rB sB tB vB hB aB bB wB xB","34":"lB"},G:{"33":"undefined lB cB I e J E F G A B C K L D gB tB AC CC"},P:{"33":"undefined OC gB YC ZC bC bB cC IC dC eC"},I:{"33":"0 1 2 3 4 5 6 7 8 9 undefined lB cB I x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T U V W X Y Z a b c R d H fB pB PC SC"}},B:NaN,C:":autofill CSS pseudo-class"};
```

Comment:
Strange that we have `undefined` in the packed array and `NaN`.

Seems like we are doing something wrong.

I am not very familiar with `packager` code, but you can try to put `console.log` or `debugger` to https://github.com/browserslist/caniuse-lite/blob/main/src/packer/feature.js#L80

and to https://github.com/browserslist/caniuse-lite/blob/main/src/packer/feature.js#L99

Reply:
One of the error could created by this line https://gist.github.com/lukewarlow/65d234b378e57b9153421b6ba65da188#file-output-js-L669

Reply:
I believe the NaN is caused by the lack of a "status" value in the caniuse formatted data. It's rather hard to determine the status of the spec for an MDN entry because it only has a `spec_url` and a `standards_track` boolean value. I wouldn't be able to map those to statuses without making some assumptions probably. I'd be curious to know what caniuse treats WICG URLs as too.

Reply:
We can use `status: "other"`

Reply:
So the undefined I think is coming from a mismatch between browser versions in MDN and browser versions in caniuse.

Reply:
The mdn data should now only generate an entry for a version where the version matches a caniuse version. And should have a status of other